build_mt.py

```diff
@@ -177,38 +177,29 @@ def correct_inds(halo_ids, N_halos_slabs, slabs, inds_fn):
 
     return ids
 
-def get_mt_info(fns, fields, minified, inds_fn):
+def get_mt_info(fns_load, fields, minified):
```

Comment:
The `get_mt_info()` function doesn't used the inds except to index the fns, so it seemed better to reduce the number of arguments and clarify the flow of information.